war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -8,8 +8,10 @@ var bootstrap = require('bootstrap-detached');
 var jenkins = require('./util/jenkins');
 var pluginManager = require('./api/pluginManager');
 
+window.zq = jquery.getJQuery();
+
 // Setup the dialog, exported
-var createPluginSetupWizard = function() {
+var createPluginSetupWizard = function(appendTo) {
 	// call getJQuery / getBootstrap within the main function so it will work with tests -- if getJQuery etc is called in the main
 	var $ = jquery.getJQuery();
 	var $bs = bootstrap.getBootstrap();
```

Comment:
I know this is not part of this change, but I never noticed this before. There's no need for `$` and `$bs`. `$bs` works for both as it's just a jquery instance with bootstrap glommed in on top.